### DISCUSSION TITLE
Can uppaal get all traces arriving at a location?

### DISCUSSION THREAD

**Discussion by Lqs66** [UPPAALModelChecker/UPPAAL-Meta#217] [Q&A]
Can uppaal get all traces arriving at a location and does the java api for uppaal support this?
Or does UPPAAL have access to all counter-example traces?

**Comment by thorulf4** [comment]
It is not possible to get all traces to some state, in part because there can be an infinite amount of such traces. But what do you want to use all those traces for? Maybe there is some other way of achieving it.

**Reply by Lqs66** [reply]
I'm using uppaal for reachability analysis, and I'm trying to get all the symbolic traces so I know which scenarios will reach the specified location.
